include/mbedtls/aes.h

```diff
@@ -13,6 +13,11 @@
  *          <em>ISO/IEC 18033-2:2006: Information technology -- Security
  *          techniques -- Encryption algorithms -- Part 2: Asymmetric
  *          ciphers</em>.
+ *
+ *          The AES-XTS block mode is standardized by NIST SP 800-38E
+ *          <https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-38e.pdf>
+ *          and the described in detail by IEEE P1619
```

Comment:
“the” → “is”, “by” → “in”

Reply:
"by" is OK here. I'll get rid of the extra "the".

Reply:
"by" is OK here. I've gotten rid of the extra "the".